packages/docs/src/examples/color-variables/DefaultColors.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div v-for="(value, key) in colors" :key="key" :style="`color: ${value};`">{{ key }} (--va-{{ key }}): {{value}}</div>
```

Comment:
Would be nice to use here black text color and some colored shape with border. Like VaColorIndicator, but handmade.

![image](https://user-images.githubusercontent.com/23530004/125068430-2428e500-e0be-11eb-8040-35477d0a6b77.png)